frontend/public/components/pod.jsx

```diff
@@ -140,10 +140,10 @@ const ContainerTable = ({heading, containers, pod}) => <div className="co-m-pane
         <div className="col-sm-2 col-xs-4">Name</div>
         <div className="col-sm-2 hidden-xs">Id</div>
         <div className="col-sm-2 col-xs-8">Image</div>
-        <div className="col-md-2 col-sm-2 hidden-xs">Security Scan</div>
         <div className="col-md-1 col-sm-2 hidden-xs">State</div>
         <div className="col-md-1 col-sm-2 hidden-xs">Restart Count</div>
         <div className="col-md-2 hidden-sm hidden-xs">Started At</div>
+        <div className="col-md-2 hidden-sm hidden-xs">Finished At</div>
```

Comment:
We should make one of the other col-sm-* columns larger since we're no longer using the entire 12 grid width

Reply:
I could make ID and Image columns both `col-sm-3` since those tend to be longer.